### ISSUE TITLE
Instrument and "stabilize" UtteranceQueue

### ISSUE THREAD

**Issue by zepumph** [phetsims/scenery-phet#342]
In https://github.com/phetsims/scenery-phet/issues/327 we started work on instrumenting UtteranceQueue, now, as part of phetsims/joist#445 we should finish the job here.

**Comment by zepumph** [comment]
In the above commits I uninstrumented AriaHerald, because I think we decided all of the direct interfacing should be done through utteranceQueue instead. AriaHerald still has a tandem that is passed into `enabledProperty` and `politeElementProperty`. I'm tempted to uninstrument `enabledProperty` and trade it out for UtteranceQueue getters/setters. @jessegreenberg could you please review and comment on how this should look as we move on in a stable way. How do you think clients should interface with aria-live attributes?

**Comment by jessegreenberg** [comment]
Thanks @zepumph, sounds great, changes look good to me.  I agree that these should be removed and that clients should only interface with UtteranceQueue.

**Comment by jessegreenberg** [comment]
Now that instrumentation is being removed from AriaHerald, axon/Property my not be necessary here, we should consider re-implenting AriaHerald so that it doesn't use it.

Also, as part of stabilizing UtteranceQueue, we should remove the aria-live attributes that we do not intend to use,  probably everything other than politeElement.

**Comment by jessegreenberg** [comment]
@zepumph could you please uninstrument the Properties in AriaHerald, and then maybe we can discuss or work on with https://github.com/phetsims/scenery-phet/issues/342#issuecomment-340870902 more in an a11y dev meeting.

**Comment by zepumph** [comment]
Here is what I think of as important PhET-iO features of our aria-live interface:

1. Add items to be spoken (through UtteranceQueue)
2. Get the current value of an aria-live attribute (Currently through monitoring the `AriaHerald.politeElementProperty`.)
3. Have some control over announcements (enabled muted).

**Comment by jessegreenberg** [comment]
@zepumph and I talked, he will work on instrumenting UtteranceQueue and I will work on cleaning up AriaHerald.

**Comment by zepumph** [comment]
I added a pretty full implementation to TUtteranceQueue. I didn't add state support yet, so it isn't being saved in the state. I'm not sure if we want that. 

From here we should finish up AriaHerald uninstrumentation.

**Comment by jessegreenberg** [comment]
@zepumph since work in AriaHerald involves removing Property, I can remove instrumentation  as part of that so we don't  run into merge conflicts.

**Comment by zepumph** [comment]
Sounds good to me!

**Comment by jessegreenberg** [comment]
In the above commit, I re-implemented AriaHerald and removed its usages of Property. This also removed instrumentation of AriaHerald. @zepumph is there anything else to do for this issue?

**Comment by zepumph** [comment]
This looks really good, I think that we are all set. Thanks for your help!
